src/lib/converter/context.ts

```diff
@@ -398,6 +398,28 @@ export class Context {
 
         return typeParameters;
     }
+
+    /**
+     * Typscript getFullyQualifiedName method don't always return unique string
```

Comment:
Yes, that's correct. If a declaration is not exported, it won't contain the source file path... It seems wrong to me to add it since it might conflict with a FQN created by the TS version. That said, I haven't read through everything yet so there might be a good reason for it that I'm not seeing...

Reply:
In 2 different files I was having the same named export.
And typescript wasn't providing me a unique identifier (no file path in front)
The result was bad reference link in the documentation.

Here in case the fqn doesn't contain a doc, I prefix it with the file path and it has resolved my problem